test/extended/conformance-k8s.sh

```diff
@@ -64,8 +64,6 @@ os::log::info "Running Kubernetes conformance suite for ${version}"
 # Execute OpenShift prerequisites
 # Disable container security
 oc adm policy add-scc-to-group privileged system:authenticated system:serviceaccounts
```

Comment:
you need to add anyuid too i think.

Reply:
Yeah, tested that locally and just updated. Looking for new results.